### PR TITLE
Fix and enhance storybook:pages tests

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #3085

### CODE REVIEW

**Comment by charlesBochet** [comment]
@gitstart-twenty It seems that CI jest, lint and modules-sb-tests are broken in your PR, could you take a look? I'm taking look at your PR in the meantime

**Comment by gitstart-twenty** [comment]
> @gitstart-twenty It seems that CI jest, lint and modules-sb-tests are broken in your PR, could you take a look? I'm taking look at your PR in the meantime

Added test fixes

File: packages/twenty-front/src/modules/users/components/UserProvider.tsx [hunk 1]
```diff
@@ -29,7 +32,7 @@ export const UserProvider = ({ children }: React.PropsWithChildren) => {
       setCurrentWorkspaceMember({
         ...workspaceMember,
         colorScheme: (workspaceMember.colorScheme as ColorScheme) ?? 'Light',
-      });
+      } as WorkspaceMember);
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/users/components/UserProvider.tsx:35]: same here

File: packages/twenty-front/src/pages/auth/__stories__/CreateProfile.stories.tsx [hunk 2]
```diff
@@ -14,7 +19,20 @@ const meta: Meta<PageDecoratorArgs> = {
   component: CreateProfile,
   decorators: [PageDecorator],
   args: { routePath: AppPath.CreateProfile },
-  parameters: {},
+  parameters: {
```
Comment (charlesBochet) [inline packages/twenty-front/src/pages/auth/__stories__/CreateProfile.stories.tsx:22]: @gitstart-twenty we can override mocks locally like this

File: packages/twenty-front/src/testing/decorators/PageDecorator.tsx [hunk 3]
```diff
@@ -29,31 +36,62 @@ const computeLocation = (routePath: string, routeParams: RouteParams) => {
   };
 };
 
+export const TestingObjectMetadataProvider = ({
+  children,
+}: {
+  children: JSX.Element;
+}) => {
+  const { objectMetadataItems: newObjectMetadataItems } =
```
Comment (charlesBochet) [inline packages/twenty-front/src/testing/decorators/PageDecorator.tsx:44]: why is this useful? :)

Comment (charlesBochet) [reply packages/twenty-front/src/testing/decorators/PageDecorator.tsx:44]: I'm trying to remove it and to use ObjectMetadataItemsProvider instead

**Review by charlesBochet — COMMENTED** [review]
I've left comments, and I'm pushing related fixes

**Comment by charlesBochet** [comment]
@gitstart-twenty So, overall, in pages storybook tests we should only rely on graphql mocks and not try to override things with useEffect

In modules storybook tests we will prefer relying on local overrides and avoid calling graphql if possible!

**Comment by charlesBochet** [comment]
I'm re-enabling the pages tests in the CI

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Comment by charlesBochet** [comment]
There is work remaining to have pages filled with data and our mini server mock to be back but this can be a story for another PR.

**Comment by gitstart-twenty** [comment]
> There is work remaining to have pages filled with data and our mini server mock to be back but this can be a story for another PR.

Okay @charlesBochet We can do the remaining work in another PR.

**Pull request merged**